SOURCE/set_PressureTemplate.c

```diff
@@ -0,0 +1,454 @@
+// =========================================================================
+// MDOODZ - Visco-Elasto-Plastic Thermo-Mechanical solver
+//
+// Copyright (C) 2018  MDOODZ Developper team
+//
+// This file is part of MDOODZ.
+//
+// MDOODZ is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// MDOODZ is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with MDOODZ.  If not, see <http://www.gnu.org/licenses/>.
+// =========================================================================
+
+#include "stdio.h"
+#include "stdlib.h"
+#include "string.h"
+#include "math.h"
+#include "header_MDOODZ.h"
+
+/*--------------------------------------------------------------------------------------------------------------------*/
+/*------------------------------------------------------ M-Doodz -----------------------------------------------------*/
+/*--------------------------------------------------------------------------------------------------------------------*/
+
+// SIMPLE
+void BuildInitialTopography( surface *topo, markers *topo_chain, params model, grid mesh, scale scaling ) {
+    
+    int k;
+    double TopoLevel = 0.0e3/scaling.L; // sets zero initial topography
+    double A = model.zmax/2.0;
+    double s = model.zmax/4.0;
+
+    for ( k=0; k<topo_chain->Nb_part; k++ ) {
+        topo_chain->z[k]     = A*exp(-pow(topo_chain->x[k],2) / 2.0/s/s  );// TopoLevel;
+        topo_chain->phase[k] = 0;
+    }
+    
+    printf( "Topographic chain initialised with %d markers\n", topo_chain->Nb_part );
+}
+
+/*--------------------------------------------------------------------------------------------------------------------*/
+/*------------------------------------------------------ M-Doodz -----------------------------------------------------*/
+/*--------------------------------------------------------------------------------------------------------------------*/
+void SetParticles( markers *particles, scale scaling, params model, mat_prop *materials  ) {    int np;
+    FILE *read;
+    int s1, s2;
+    
+    // Define dimensions;
+    double Lx = (double) (model.xmax - model.xmin) ;
+    double Lz = (double) (model.zmax - model.zmin) ;
+    double T_init = (model.user0 + zeroC)/scaling.T;
+    double P_init = model.PrBG;
+    double setup  = (int)model.user1;
+    double radius = model.user2/scaling.L;
+    double X, Z, Xn, Zn, xc = 0.0, zc = 0.0, sa=radius/2.0, la=radius*2.0, theta=-30.0*M_PI/180.0;
+    double Ax, Az;
+
+    // Fixed random seed
+    srand(69);
+
+    char *ph_hr;
```

Comment:
Could you please rename it?
Phases1DArray, but maybe there is more accurate description for this type of data structure

Reply:
This was removed